docs/UsersGuide/source/Tutorial.rst

```diff
@@ -0,0 +1,589 @@
+
+Users can run through the entire set of tutorials or jump to the one that interests them most. The five tutorials address different skills:
+
+   #. :ref:`Severe Weather Over Indianapolis <fcst1>`: Change physics suites and compare graphics plots. 
+   #. :ref:`Cold Air Damming <fcst2>`: Coming soon!
+   #. :ref:`Southern Plains Winter Weather Event <fcst3>`: Coming soon!
+   #. :ref:`Halloween Storm <fcst4>`: Coming soon!
+   #. :ref:`Hurricane Barry <fcst5>`: Coming soon!
+
+Each section provides a summary of the weather event and instructions for configuring an experiment. 
+
+.. _fcst1:
+
+Sample Forecast #1: Severe Weather Over Indianapolis
+=======================================================
+
+**Objective:** Modify physics options and compare forecast outputs for similar experiments using the graphics plotting task. 
+
+Weather Summary
+--------------------
+
+A surface boundary associated with a vorticity maximum over the northern Great Plains moved into an unstable environment over Indianapolis, which led to the development of isolated severe thunderstorms before it congealed into a convective line. The moist air remained over the southern half of the area on the following day. The combination of moist air with daily surface heating resulted in isolated thunderstorms that produced small hail. 
+
+**Weather Phenomena:** Numerous tornado and wind reports (6/15) and hail reports (6/16)
+
+   * `Storm Prediction Center (SPC) Storm Report for 20190615 <https://www.spc.noaa.gov/climo/reports/190615_rpts.html>`__ 
+   * `Storm Prediction Center (SPC) Storm Report for 20190616 <https://www.spc.noaa.gov/climo/reports/190616_rpts.html>`__
+
+.. figure:: _static/IndySevereWeather18z.gif
+   :alt: Radar animation of severe weather over Indianapolis on June 15, 2019 starting at 18z. The animation shows areas of heavy rain and tornado reports moving from west to east over Indianapolis. 
+
+   *Severe Weather Over Indianapolis Starting at 18z*
+
+Data
+-------
+
+On `Level 1 <https://github.com/ufs-community/ufs-srweather-app/wiki/Supported-Platforms-and-Compilers>`__ systems, users can find data for the Indianapolis Severe Weather Forecast in the usual input model data locations (see :numref:`Section %s <DataLocations>` for a list). The data can also be downloaded from the `UFS SRW Application Data Bucket <https://noaa-ufs-srw-pds.s3.amazonaws.com/index.html>`__. 
+
+   * FV3GFS data for the first forecast (``control``) is located at: 
+   
+      * https://noaa-ufs-srw-pds.s3.amazonaws.com/index.html#input_model_data/FV3GFS/grib2/2019061518/
+
+   * HRRR and RAP data for the second forecast (``test_expt``) is located at: 
+      
+      * https://noaa-ufs-srw-pds.s3.amazonaws.com/index.html#input_model_data/HRRR/2019061518/
+      * https://noaa-ufs-srw-pds.s3.amazonaws.com/index.html#input_model_data/RAP/2019061518/
+
+Load the Regional Workflow
```

Comment:
Since the `regional_workflow` repo is now retired, we should probably rename the environment to `srw_app` or something similar.  Something to talk about at the next code management meeting... (nothing needs to change here though).